check-dependencies.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+# flake8: noqa
```

Comment:
Could you narrow this one down with the specific rule to ignore? You can do with `# noqa: E123

Reply:
In this file it's just a bunch of "F401 imported but unused" (which is the intention of this script). You can't ignore a specific rule for an entire file, so I'll have to add  `# noqa: F401` to each line. But I just remembered there's another trick using `__all__` which may or may not be cleaner :)